src/common/util.js

```diff
@@ -417,8 +417,16 @@ const util = {
     }
 
     return true
-  }
+  },
 
+  camelize(str) {
```

Comment:
Let's call it `camelcase` because we already have `downcase` and `titlecase` helpers.